docker/test/integration/features/python.feature

```diff
@@ -174,3 +174,13 @@ Feature: MiNiFi can use python processors in its flows
 
     Then the Minifi logs contain the following message: "key:error.message value:Error" in less than 60 seconds
     And the Minifi logs contain the following message: "key:my.attribute value:my.value" in less than 10 seconds
+
+  @USE_NIFI_PYTHON_PROCESSORS
+  Scenario: MiNiFi C++ can use native NiFi source python processors
```

Comment:
You forgot to update the name of the scenario.

Reply:
Good catch! It's strange because I thought behave doesn't allow duplicate scenario names, but it didn't complain. Updated in 5f47d5cc2cfe751eca386d8be9ba1b9657cf5255